Userland/Libraries/LibGL/GLContext.cpp

```diff
@@ -2193,6 +2231,41 @@ void GLContext::gl_draw_elements(GLenum mode, GLsizei count, GLenum type, const
     gl_end();
 }
 
+void GLContext::gl_array_element(GLint i)
+{
+    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_array_element, i);
+    RETURN_WITH_ERROR_IF(i < 0, GL_INVALID_VALUE);
+
+    // This is effectively the same as `gl_draw_elements`, except we only output a single
+    // primitive (this is done between a `gl_begin/end` call) that is to be rendered.
+    if (!m_client_side_vertex_array_enabled)
+        return;
+
+    if (m_client_side_color_array_enabled) {
+        float color[4] { 0, 0, 0, 1 };
+        read_from_vertex_attribute_pointer(m_client_color_pointer, i, color, true);
```

Comment:
These calls to `read_from_vertex_attribute_pointer` no longer have that last boolean argument in the latest `master` branch